modules/mail/mod_settings/config.xml

```diff
@@ -41,6 +41,18 @@
     		<type>string</type>
     		<default>localhost</default>
     		<description>Hostname des SMTP-Servers</description>
+    	</item>
+		<item>
+    		<name>mail_smtp_port</name>
+    		<type>int</type>
+    		<default>25</default>
```

Comment:
I would vote for going with Port {{465}} and TLS enabled as standard config.

Reply:
This sounds like a good approach!
Let me look into this, once I got back into testing.

Reply:
resolved via 48a24ae4f6f4597f00d03925537a0e816341671b